docs/DeveloperGuide.md

```diff
@@ -44,3 +47,4 @@ Ideas and structure for the User Guide: https://se-education.org/addressbook-lev
 ## Instructions for manual testing
 
 {Give instructions on how to do a manual product testing e.g., how to load sample data to be used for testing}
+- TODO: Talk about how to test the with different kinds of JSON file, and provide a sample JSON file with instructions on how to use it. (SX)
```

Comment:
Missing word after "how to test the ... "

Reply:
Implemented in c34bb141b0dc22f370cd3e6f1d641220077ecbad